FreydCategoriesForCAP/gap/AdditiveClosure.gi

```diff
@@ -151,6 +185,37 @@ InstallMethod( \/,
     
 end );
 
+##
+InstallMethod( \/,
+          [ IsCapCategoryObject, IsAdditiveClosureCategory ],
+  function( o, C )
+    
+    if not IsIdenticalObj( C, AdditiveClosure( CapCategory( o ) ) ) then
```

Comment:
cheaper: use `UnderlyingCategory( C )` and compare it with `CapCategory( o )`

Reply:
I have already adjusted the code.